yapapi/executor/__init__.py

```diff
@@ -314,10 +314,16 @@ async def worker_starter() -> None:
 
             self.emit(events.ComputationFinished(job.id))
 
-        except (Exception, CancelledError, KeyboardInterrupt):
+        except (Exception, CancelledError, KeyboardInterrupt) as e:
+            #   TODO: why do we catch KeyboardInterrupt? How can we get one here?
             self.emit(events.ComputationFinished(job.id, exc_info=sys.exc_info()))  # type: ignore
             cancelled = True
 
+            if isinstance(e, CancelledError):
```

Comment:
@johny-b I wonder - wouldn't we want to re-raise any exception here? ... I mean, why are we consuming `Exception` in this block?  ...

Reply:
Maybe here we might get some internal exception here, like "some API returned 500 because the network was down" and we might not want to propagate this to the part that uses Golem?
But that's just a guess. I only know that in some form this was here in many earlier versions.